container/runtime/src/main/java/org/wildfly/swarm/container/runtime/RuntimeServer.java

```diff
@@ -23,7 +23,14 @@
 import java.lang.reflect.Method;
```

Comment:
It doesn't appear that any of the changes in RuntimeServer are relevant to this fix, is that correct?